vendor/k8s.io/kubernetes/test/utils/image/manifest.go

```diff
@@ -277,6 +354,23 @@ func ReplaceRegistryInImageURL(imageURL string) (string, error) {
 	countParts := len(parts)
 	registryAndUser := strings.Join(parts[:countParts-1], "/")
 
+	if repo := os.Getenv("KUBE_TEST_REPO"); len(repo) > 0 {
```

Comment:
comment for what this is trying to do?  I'm not clean on why it is conditional and why it returns early instead of updating the repo and falling through.

Reply:
If you don't specify the env var, you get the existing behavior.  This is an optional env var that overrides the current "per repo" mapping.